### PR TITLE
Extract markdown from source-browsing fixture

### PR DESCRIPTION
**Pull request by sebastinez**
- Extracts markdown files from `source-browsing` fixture
- Renames variables to differentiate between fixtures (we'll have eventually more like cobs, reviews, etc.)
- Some refactoring in globalSetup to get multiple fixtures running.
- Adds markdown footnotes rendering test

### CODE REVIEW

**Comment by rudolfs** [comment]
We should do a full screen screenshot for the footnotes (the 2nd footnote is cut off otherwise) and for some reason the black square isn't rendered properly:

Screenshot 2023-06-02 at 14 33 41

**Review by rudolfs — COMMENTED** [review]
Nice! One thought: could we instead of having two peers "palm" and "john" just host the two project fixtures on one node?

**Comment by sebastinez** [comment]
> We should do a full screen screenshot for the footnotes (the 2nd footnote is cut off otherwise) and for some reason the black square isn't rendered properly:
> 
> Screenshot 2023-06-02 at 14 33 41

Ok did a scrolling down to footnotes, so we see the entire note and do a fullscreen.
Palm is now hosting both repos. (markdown and source-browsing)
And the black square is back, was an issue with the specified path

File: tests/visual/markdown.spec.ts [hunk 1]
```diff
@@ -168,7 +168,7 @@ test.describe("markdown rendering", async () => {
   });
 });
 
-test("relative image not able to being loaded", async ({ page }) => {
+test.only("relative image not able to being loaded", async ({ page }) => {
```
Comment (rudolfs) [inline tests/visual/markdown.spec.ts:171]: 👀

File: tests/support/fixtures.ts [hunk 2]
```diff
@@ -304,14 +297,42 @@ export async function createSeedFixture() {
   );
 }
 
-export const aliceMainHead = "fcc929424b82984b7cbff9c01d2e20d9b1249842";
+export async function createMarkdownFixture(palm: RadiclePeer) {
```
Comment (rudolfs) [inline tests/support/fixtures.ts:300]:
```suggestion
export async function createMarkdownFixture(peer: RadiclePeer) {
```

File: tests/unit/mutexExecutor.test.ts [hunk 3]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-floating-promises */
```
Comment (rudolfs) [inline tests/unit/mutexExecutor.test.ts:1]: This actually changes how the test works if I'm not mistaken. We should keep it as is.

File: tests/visual/markdown.spec.ts [hunk 4]
```diff
@@ -82,9 +82,30 @@ test.describe("markdown rendering", async () => {
   test.describe(async () => {
     test.use({ viewport: { width: 1280, height: 590 } });
```
Comment (rudolfs) [inline tests/visual/markdown.spec.ts:83]: We should probably remove this if we're taking a full-screen screenshot below.

**Review by rudolfs — APPROVED** [review]

**Pull request merged**